everestrs/tests/types/example.yaml

```diff
@@ -0,0 +1 @@
+e
```

Comment:
@dorezyuk That seems random?

Reply:
yes true - will place something more descriptive here